### ISSUE TITLE
Downloading package with wrong options

### ISSUE THREAD

**Issue by lasers** [rmarquis/pacaur#580]
I was trying to download package only with pacaur.
```
chris@z420 ~ $ pacaur -d community/ttf-ubuntu-font-family
:: Retrieving package(s)...
sed: -e expression #1, char 46: unexpected `}'
```
I was wrong and really wanted `pacaur -Sw ttf-ubuntu-font-family`.

Catch that error? It does same thing for `pacaur -d aur/some-package-name`.  However, `pacaur -d some-package-name` without `aur/` will work perfect. Improvements?

**Comment by rmarquis** [comment]
The inclusion of the `repo/` syntax makes little sense here, since it will only work for AUR package. Might be worth adding when pacman gets support for ABS.

**Comment by lasers** [comment]
I type some. I shift+click some (usually just long names). When shift+clicking, it will highlight the `repo/package` for a quick paste. I was curious to see if this error would occur too with aur package (it did). I made this issue to see if pacaur could take care of `aur/packages` with `-d`. (Possibly other aur options too).

While I'm here, I wanted to ask something about `-a, --aur`. You can use search, download, build, install or update target(s) from the AUR. Would it be possible / good feature to add list to the list too?

Typing `pacaur -Qa` would list all installed packages from AUR.

Typing `pacaur -Qsa vim` would only list the installed packages from AUR like...
```
pacaur -Ssa vim | grep -iA1 --no-group-separator "\[installed"
```

There might be some other possibilities too.

**Comment by rmarquis** [comment]
I see. The `-d` option is currently a weird beast, since it has no pacman equivalent, but will be eventually merged with the upcoming `pacman -B`.

> Would it be possible / good feature to add list to the list too?

AUR packages are considered foreign packages by pacman and can be already listed with `-Qm`. In fact, in the deeper level pacaur exclusively works with that "foreign" concept. If the package can't be found in the repositories (and thus foreign), then it looks for it in the AUR.

**Comment by lasers** [comment]
I love Arch for AUR packages and it have been while since I touch pacman. Since I have been in relationship with pacaur longer than any other AUR helpers, could pacaur please support that feature?

The `-a` and `-r` options are only pacman extension options and is exclusive to pacaur. I use them both all times to get clean/shorter queries when I'm finding something new and exciting to install. I would like for the same feature to be there next time I'm finding something old or foreign to uninstall.

Looking closely at three things `-a, --aur` and `-r, --repo` and at `you` because they are the heart of pacaur. Like you said, the two options are there for searching, downloading, building, installing or updating targets from the AUR (or the repositories), but not querying. It might be the last extended pacman/pacaur option to add.

Here's what I see on my terminal.

`pacaur -Q` is for pacman.................................................Aur+Repo.
`pacaur -Qm` is for pacaur+foreign...............................No-Repo.
`pacaur -Qa` should be for pacaur+foreign only..........No-Repo.    <---- Fix?
`pacaur -Qr` is for pacman...............................................No-Aur.

--

`pacaur -Qs` is for pacman + pacaur / foreign..............Aur+Repo.
`pacaur -Qsa` should be for pacaur+foreign only.........No-Repo.    <------ Fix?
`pacaur -Qsr` should be for pacman only........................No-AUR.    <------ Fix? 

I understand some options does not do anything... Just getting myself a bit more familiar with pacaur by trying new options or commands... and comparing stuffs. :-)

**Comment by rmarquis** [comment]
There are no "AUR" and "Repo". There are sync, local databases and the AUR. The option `-Q` is for local only,

Query local packages as the AUR doesn't make sense. Pacaur is built upon pacman/makpekg, not the other way around. true, there is `-Qu` which checks update against the sync database, so `-Qua` was added. But if there is something to fix here, that would be to change it to `-Qum`.

**Comment by lasers** [comment]
Okay. Thanks for answering everything and being helpful. I love pacaur. You have a good day. You may close this if you want.

**Comment by rmarquis** [comment]
You are welcome.

Your post convinced me that I should replace `-Qur` by `-Qun` (for `--native`) and `-Qua` by `-Qum` (for `--foreign`). The `-Qu` shortcut will still display both, as it is the case as of now.

**Comment by rmarquis** [comment]
The `-d` with wrong option issue is fixed in 063cf391f92a701a6e3f1effba47014b007090dd. You can also use the `aur/` prefix too.
